app/models/deploy_queue.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class DeployQueue
+  include Singleton
+
+  STAGGER_INTERVAL = 10.seconds
```

Comment:
most likely want this ENV configureable

Reply:
default should be off, small deployments don't need this

Reply:
You mean this env would be set up on a per project/stage basis?

Reply:
samson env